pkg/csi/service/vanilla/controller.go

```diff
@@ -55,8 +56,9 @@ type NodeManagerInterface interface {
 }
 
 type controller struct {
-	manager *common.Manager
-	nodeMgr NodeManagerInterface
+	manager           *common.Manager
+	nodeMgr           NodeManagerInterface
+	coCommonInterface commonco.COCommonInterface
```

Comment:
I think we can keep an instance of COCommonInterface, just like volumeMigrationService.
We do not need this to be part of  controller struct.

name `commonco.COCommonInterface` is not looking good and easy to read. Can refactor make it better in next PR?